reach

```diff
@@ -1155,16 +1345,14 @@ do_server () {
   fi
   export REACH_RPC_PORT
 
-  # TODO: auto-gen self-signed tls cert if not present
-  TLS_PATH='reach-server'
   if [ "x${REACH_RPC_TLS_KEY}" = "x" ]; then
-    REACH_RPC_TLS_KEY=${TLS_PATH}.key
+    REACH_RPC_TLS_KEY=${TLS_SAN}.key
   fi
   if [ "x${REACH_RPC_TLS_CRT}" = "x" ]; then
-    REACH_RPC_TLS_CRT=${TLS_PATH}.crt
+    REACH_RPC_TLS_CRT=${TLS_SAN}.crt
   fi
   if [ "x${REACH_RPC_TLS_PASSPHRASE}" = "x" ]; then
-    REACH_RPC_TLS_PASSPHRASE=rpc-demo
+    REACH_RPC_TLS_PASSPHRASE="$TLS_DEFAULT_PASSPHRASE"
   fi
   export REACH_RPC_TLS_KEY REACH_RPC_TLS_CRT REACH_RPC_TLS_PASSPHRASE
 
```

Comment:
Does it die if these files don't exist? If not, it should. Ideally, if the ENV vars aren't set and the files don't exist, then it should automatically `do_tls`

Reply:
The script didn't die before, but the server has logic to handle that case.

629a8be39b206769a14c79dcf16af93d82091cbd addresses this.